src/Uno.UI/Controls/NSFontHelper.macOS.cs

```diff
@@ -17,17 +17,11 @@ namespace Windows.UI
 {
 	internal static class NSFontHelper
 	{
-		private static Func<nfloat, FontWeight, FontStyle, FontFamily, nfloat?, NSFont> _tryGetFont;
+		private static Func<nfloat, FontWeight, FontStyle, FontFamily, nfloat?, NSFont> _tryGetFont = ((Func<nfloat, FontWeight, FontStyle, FontFamily, nfloat?, NSFont>)InternalTryGetFont).AsMemoized();
```

Comment:
I think we can use this instead:

```suggestion
		private static Func<nfloat, FontWeight, FontStyle, FontFamily, nfloat?, NSFont> _tryGetFont = Funcs.Create(InternalTryGetFont).AsMemoized();
```